uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/FeaturesServerTest.java

```diff
@@ -72,7 +72,7 @@ public void clean() {
 	 */
 	@Test
 	public void testFuseJbossRuntimeDetection() {
-		downloadRuntime("JBoss Fuse 6.3.0");
+		downloadRuntime("Red Hat Fuse 6.3.0");
```

Comment:
need to be kept as-is for 6.x https://issues.jboss.org/browse/FUSETOOLS-2843?focusedCommentId=13545002&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13545002